fuzzing/tools/BUILD

```diff
@@ -15,10 +15,43 @@
 
 # This package contains tool scripts
 
-load("@rules_python//python:defs.bzl", "py_binary")
+load("@rules_python//python:defs.bzl", "py_binary", "py_library", "py_test")
+
+py_library(
+    name = "dict_validator_lib",
+    srcs = ["dict_validator.py"],
+    visibility = ["//visibility:public"],
+)
 
 py_binary(
     name = "launcher",
     srcs = ["launcher.py"],
     visibility = ["//visibility:public"],
 )
+
+py_binary(
+    name = "dict_validator",
+    srcs = ["dict_validator.py"],
+    visibility = ["//visibility:public"],
+)
+
+py_test(
+    name = "dict_validator_test",
+    srcs = ["dict_validator_test.py"],
+    data = [
+        "dict_data/invalid.dict",
+        "dict_data/valid.dict",
+    ],
+    deps = [
+        ":dict_validator_lib",
+        "@bazel_tools//tools/python/runfiles",
+    ],
+)
+
+py_test(
+    name = "dict_validator_unit_test",
+    srcs = ["dict_validator_unit_test.py"],
+    deps = [
+        ":dict_validator_lib",
+    ],
+)
```

Comment:
It doesn't look like we need both of these - let's consolidate our testing in the API unit test.